pkg/collector/corechecks/oracle-dbm/statements.go

```diff
@@ -444,8 +444,8 @@ func (c *Check) StatementMetrics() (int, error) {
 			 */
 			err = c.db.Select(
 				&DDForceMatchingSignatures,
-				"SELECT distinct force_matching_signature FROM v$sqlstats WHERE sql_text like '%/* DD%' and (sysdate-last_active_time)*3600*24 < :1",
-				time.Since(c.statementsLastRun).Seconds(),
+				"SELECT distinct force_matching_signature FROM v$sqlstats WHERE sql_text like '%/* DD%' and (sysdate-last_active_time)*3600*24 <= :1",
+				time.Since(c.statementsLastRun).Seconds()+1,
```

Comment:
sorry how is the timeout being set? It's not clear to me

Reply:
closing this and oponing a new one. i picked a wrong branch